lib/package/plugins/TD007-targetSslInfo-Truststore.js

```diff
@@ -79,8 +79,8 @@ const onTargetEndpoint = function (endpoint, cb) {
       messages.forEach((message) => {
         endpoint.addMessage({
           plugin,
-          line: htc.getElement().lineNumber,
-          column: htc.getElement().columnNumber,
+          line: sslInfos[0].lineNumber,
```

Comment:
is this correct? Shouldnt there be a null check for sslInfos?

Reply:
No, not correct. It should be simplified.